recipes/mplhep/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mplhep" %}
+{% set version = "0.1.14" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b1a094fe352cc98ef5480c8b4a3bc12d75088b3fd7ae00f1582043838d4a8369
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib >=3.2
+    - numpy >=1.16.0
+    - packaging
+    - python
+    - requests >=2.21,<3
+    - scipy >=1.1.0
+
+test:
+  imports:
+    - mplhep
+
+about:
+  home: "https://github.com/scikit-hep/mplhep"
+  license: MIT
+  license_file: LICENSE
+  summary: "Matplotlib styles for HEP"
+  doc_url: ""
```

Comment:
This still counts as being empty, if you don't want to put a URL here for whatever reason it should be removed entirely:
```suggestion
```

Reply:
Sorry, I misunderstood the previous comment.